build.py

```diff
@@ -15,7 +15,7 @@
 
 if Configuration.current.target.sdk == OSType.Linux:
 	foundation.CFLAGS = '-DDEPLOYMENT_TARGET_LINUX -D_GNU_SOURCE '
-	foundation.LDFLAGS = '${SWIFT_USE_LINKER} -Wl,@./CoreFoundation/linux.ld -lswiftGlibc `icu-config --ldflags` -Wl,-defsym,__CFConstantStringClassReference=_TMC10Foundation19_NSCFConstantString -Wl,-Bsymbolic '
+	foundation.LDFLAGS = '${SWIFT_USE_LINKER} -Wl,@./CoreFoundation/linux.ld -lswiftGlibc `pkg-config icu-uc icu-i18n --libs` -Wl,-defsym,__CFConstantStringClassReference=_TMC10Foundation19_NSCFConstantString -Wl,-Bsymbolic '
```

Comment:
I don't think that's necessary - pkg-config isn't architecture-specific; it relies on .pc text files it finds in the `PKG_CONFIG_PATH`. I also have the prefixed version on my Raspberry Pi, but it's just a symlink to unprefixed `pkg-config`. It's not required for cross-compiling either; I've been cross-compiling for the RPi using the `pkg-config` I got from OSX homebrew. It just reads the .pc files from the RPi sysroot, and I've cross-compiled basically all of the swift tools like that.